app/src/main/kotlin/com/wire/android/ui/calling/incoming/IncomingCallViewModel.kt

```diff
@@ -78,14 +80,16 @@ class IncomingCallViewModel @Inject constructor(
         }
     }
 
-    private fun observeConversationDetails(conversationDetails: ConversationDetails) {
-        val conversationName = when (conversationDetails) {
-            is ConversationDetails.Group -> conversationDetails.conversation.name
-            is ConversationDetails.OneOne -> conversationDetails.otherUser.name
-            else -> null
+    private fun initializeScreenState(conversationDetails: ConversationDetails) {
+        callState = when (conversationDetails) {
+            is ConversationDetails.Group -> callState.copy(conversationName = conversationDetails.conversation.name)
+            is ConversationDetails.OneOne -> {
+                callState.copy(
+                    conversationName = conversationDetails.otherUser.name,
+                    avatarAssetId = UserAvatarAsset(conversationDetails.otherUser.completePicture ?: "")
```

Comment:
Wouldn't be easier to handle the user avatar later if we check if it is just null or not instead of checking if it is null, empty or something else?